articles/sentinel/process-events-normalization-schema.md

```diff
@@ -187,7 +187,7 @@ The process event schema references the following entities, which are central to
 | **ParentProcessInjectedAddress**   | Optional     | String     |    The memory address in which the responsible parent process is stored.           |
 | **ParentProcessId**| Mandatory    | integer    | The process ID (PID) of the parent process.   <br><br>     Example:  `48610176`    |
 | **ParentProcessGuid**              | Optional     | String     |  A generated unique identifier (GUID) of the parent process.     <br><br> Example: `EF3BD0BD-2B74-60C5-AF5C-010000001E00` |
-| **ParentProcessIntegrityLevel**    | Optional     | String     |   The integrity level of the parent process. Windows assigns integrity levels to processes based on certain characteristics, such as if they were launched from an internet download. <br><br>Assigned integrity levels influence permissions to access resources. |
+| **ParentProcessIntegrityLevel**    | Optional     | String     |   Every process has an integrity level that is represented in its token. Integrity levels determine the process level of protection or access. <br><br> Windows defines four integrity level - low, medium, high, and system. Standard users receive medium, elevated users receive high. <br><br> For mode information refer to [Mandatory Integrity Control - Win32 apps](https://docs.microsoft.com/en-us/windows/win32/secauthz/mandatory-integrity-control). |
```

Comment:
@oshezaf  - see the edit above ("the following" instead of "four", bolding on the integrity level names and other changes in that sentence, and "more" instead of mode")